lib/src/test/java/com/thenewboston/utils/BankApiMockEngine.kt

```diff
@@ -40,27 +40,37 @@ class BankApiMockEngine {
             it.addHandler { request ->
                 when (request.url.encodedPath) {
                     BankAPIJsonMapper.ACCOUNTS_ENDPOINT -> {
-                        val content = BankAPIJsonMapper.mapAccountsToJson()
+                        val offset = request.url.parameters.get("offset")!!.toInt()
+                        val limit = request.url.parameters.get("limit")!!.toInt()
```

Comment:
Instead of !! just use safe and elvis for handling nullable types. 

request.url.parameters.get("offset")?.toInt() ?: 0

Reply:
done